packages/client/bin/devnet.json

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
I don't love this file in this folder, but maybe for clarity we can rename it to `devnet-chain.json`

Reply:
I need to look at the `parse` functions again.  I may be able to slim down the genesis parameters and just construct an object  instead of the extra file.  I just needed something for generating the genesis block stateRoot

Reply:
I've replaced the external json file with a custom params object.  Let me know if you have an alternative you'd prefer.